pkg/thirdparty/ipfs/go-ipfs/README.md

```diff
@@ -0,0 +1,4 @@
+## github.com/ipfs/go-ipfs
```

Comment:
are these packages intended to be exposed outside the repo?

we have used a bit of a different folder convention in other repos.

Reply:
internal?

Reply:
Yes I would prefer an internal path for third-party copied packages where we can.

Reply:
I'll rename the copied package to /internal/ipfs/go-ipfs/core/coredag